### ISSUE TITLE
Bug: Interface moves up, making it unusable

### ISSUE THREAD

**Issue by zarazandra** [DarKDinDoN/theme-glass#5]
Hi there, 

I love the theme, because it is so minimalistic. 
Problem is: the whole game slips upward a bit, so that I can't use the menu for chat, actor, scenes etc - they're not reachable. Reloading helps, but as soon as someone rolls a dice OR clicks on the chat-icon top right, everything moves up. 

normal: 
![image](https://github.com/DarKDinDoN/theme-glass/assets/70195103/ab9315ab-c5f8-4395-8624-72b4e3b3a677)

After clicking the chat-symbol: 
![image](https://github.com/DarKDinDoN/theme-glass/assets/70195103/cb47cba9-a861-4f34-bf0b-3b19c15b6177)

I have no clue, what causes this, but maybe you have? 

I play with the TDE/DSA5-ruleset with no internal skin added.

**Comment by mech-tools** [comment]
Hi,

I couldn't reproduce the issue with TDE/DSA5 and the internal skin disabled.

I can think of two possible issues here:
- Another module is responsible for this issue. Please try to disable all modules except Theme: Glass.
If you can't reproduce this issue will all other modules disabled, then one of the disabled module has a conflict with Theme: Glass. In this case, try to enable each module one by one until you find the conflicting module.
- You have a specific layout with Theme: Glass, please provide your components configuration ("Hide interface components" window within Theme: Glass settings)

**Comment by zarazandra** [comment]
Hi there, 

for the first point: have to check later/tomorrow, but I tried to manage to fix it with custom css; maybe that helps with the problem? 

-> 
```
#interface {
	overflow: visible;
}
```

//Edit: managed to look right now. Taskbar (from theripper93) makes trouble. 

for the second point: 

![image](https://github.com/DarKDinDoN/theme-glass/assets/70195103/0c920f07-f4e5-459e-af9f-2367eb4c1e21)
![image](https://github.com/DarKDinDoN/theme-glass/assets/70195103/24f299e6-05a2-4876-ae06-a9927ed4b188)

**Comment by zarazandra** [comment]
And, for another thing - to be honest, I didn't redo the changes to check, if it's vanilla, but maybe you can check this from your side; 
I have found a thing inside the compendia. 
The font color is set to black, on a dark background. I changed the font color inside the CSS-file of yours to white (didn't manage to do that with custom css).  
![image](https://github.com/DarKDinDoN/theme-glass/assets/70195103/9b692830-e1e7-4f1f-9c8f-7f934cffca6c)
Found here: 
![image](https://github.com/DarKDinDoN/theme-glass/assets/70195103/8a26766f-b866-4a3f-8ddb-95e6b149ab1d)
(class: create-button create-folder).

**Comment by mech-tools** [comment]
Hi,

I'll check the "Taskbar" module compatibility later.

For the color in the compendiums, it seems that the TDE/DSA5 system adds CSS styles even tho the internal theme is deactivated.
Try this within Custom CSS:
```css
.sidebar-tab .directory-list .directory-item h3 {
    color: inherit;
}
```

**Comment by mech-tools** [comment]
Hi,

Regarding "TaskBar" module compatibility: I might be wrong here, but the module seems to be behind a paywall (patreon).
If not, can you provide me with the link to the module?

**Comment by zarazandra** [comment]
Hi,

yes, it's a premium module.
I would ask the creator if I can share it with you for debugging purposes.

**Comment by zarazandra** [comment]
I already got answer: 

> theripper93 — today at 14:35 Uhr
> Any developer that needs one or more of my premium modules for compatibility\integration can DM me and i'll send them what they need free of charge 🙂
> 

You can reach him via discord or patreon and, for sure, github under the same handle. 


P.S.: Custom CSS worked like a charm! Thank you!

**Comment by mech-tools** [comment]
Ok, TaskBar is installed and activated.
I still can't reproduce the issue. But it might be because I have a different configuration than your for TaskBar.

- Can you send me your TaskBar module configuration
- A screenshot of the entire FVTT application, so I can see how TaskBar is configured (e.g: buttons on the right of the TaskBar)

**Comment by zarazandra** [comment]
sure thing: 

![image](https://github.com/DarKDinDoN/theme-glass/assets/70195103/1cfec248-0a77-448d-9da5-5d436c7d3c4c)
![image](https://github.com/DarKDinDoN/theme-glass/assets/70195103/32aa54e9-be04-4d72-894f-9c3eaf17615e)

**Comment by mech-tools** [comment]
I believe this is fixed with:  [1.6.2](https://github.com/DarKDinDoN/theme-glass/releases)
